### ISSUE TITLE
Playback Recommendations?

### ISSUE THREAD

**Issue by weaverm** [donmelton/other_video_transcoding#157]
I have Plex hosting my videos serving (via Direct Play) the Plex client on the most recent Apple TV 4K (2021) connected via a proper HDMI cable to a 2019 LG OLED TV. My 10-bit h.265 encodes of blu-rays at the default bit rates often show color banding on my setup. So do straight rips of 4K UHDs. Things look much better on my 27" iMac screen.

There's a big thread on the Plex forums with people complaining about color banding ([https://forums.plex.tv/t/apple-tv-4k-2021-hdr-color-banding/718742](https://forums.plex.tv/t/apple-tv-4k-2021-hdr-color-banding/718742)) that's getting to be nearly a year old. One can reasonable start to wonder if it'll ever be addressed?

My question is, what do y'all use? Surely you're not just putting up with the ugly color banding like have been. Is there some different software or perhaps different hardware I could/should be using? All the streaming boxes seem to have one trade-off or another, but maybe there's something I'm overlooking?

**Comment by donmelton** [comment]
@weaverm I use a Amazon Fire TV 4K stick and everything looks fine. Keep in mind that while hardware-based 10-bit HEVC encoding will mitigate far more color banding than hardware-based 8-bit encoding, it still isn't perfect. For really difficult titles like _Blade Runner: 2049_, I recommend using the new `--x264-cbr` option in `other-transcode` for 1080p content. That, of course, isn't viable for 4K content. With 4K, you're really better off investing in more storage and just playing your rips directly.

**Comment by weaverm** [comment]
Are you using the Plex client on the Amazon Fire TV 4K stick or something else?

I don't mess with transcoding 4K stuff because it still seems like quite a hassle.

**Comment by klogg416** [comment]
Seeing banding on raw UHD rips suggests a settings problem. Quite a few of us use Fire TV 4K sticks and/or AppleTV 4Ks and don't experience banding on raw rips. As Don pointed out already, some transcodes will pick up slight banding, but it is rare. And some content comes with banding in the master which cannot be fixed.

My 2017(?) LG OLED indicates when it is switching to HDR and the type of HDR, and the picture settings on the TV also reflect extra HDR options. I would confirm that the right ones are kicking on based on the input. I would also make sure that the AppleTV is configured for the most "pure" or unadulterated output. UHD raw rips on the AppleTV should look as good as off the disc...

**Comment by weaverm** [comment]
I double checked the various settings on the TV and Apple TV. All the "features" LG puts on their TVs to "improve" the picture are turned off. Deep color is turned on. The only settings I'm aware of on the Apple TV are refresh rate and HDR format; mine is set to 60Hz 4K Dolby Vision since that's what the TV supports. I tried changing it to 'only' HDR, but it made no difference. Plex is playing via Direct Play, so no transcoding.

The most recent movie I noticed this on was my untranscoded 4K rip of Rambo: Last Blood (2019). About 90 seconds into the movie I can see color banding in the sky. I took a picture of with my cell phone, so you'll have to ignore the  moiré pattern from the screen interacting with the camera sensor.
![RamboLastBlood](https://user-images.githubusercontent.com/2437815/165393563-d4e19ea9-f51e-4603-af70-0997d33c492c.jpg)

Interestingly, I pointed Infuse on my Apple TV at my same Plex server and in the same movie I did not see any color banding in that scene. The next time I notice banding in a 10-bit h.265 transcode of a blu-ray rip, I'll try switching my client to Infuse.

**Comment by donmelton** [comment]
@weaverm Yeah, I can definitely see the color banding there. If Infuse isn't showing the same behavior then it's likely that the Gamma defaults for Infuse are somehow different than the Plex app.
